.changes/next-release/Bug Fix-335f2fb2-ad96-4031-8777-1edba11b4d21.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Fixed code block text background"
```

please review https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#changelog

the changelog should be meaningful. "fixe code block" lacks context, and has no meaning to anyone not looking directly at this PR. a meaningful entry would be something like:

```suggestion
	"description": "Amazong Q Chat: Fixed code block text background"
```

note: you can edit this file directly, no need to run `npm run newChange` again.